src/components/Navbar.tsx

```diff
@@ -0,0 +1,64 @@
+import { NavLink } from "react-router-dom";
+import { IUser } from "../utils/interfaces";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+import useFetchUsers from "../utils/useFetchUsers";
+import "./Navbar.css";
+
+interface NavProps {
+  handleUserClicked: (
+    e: React.ChangeEvent<HTMLSelectElement>,
+    usersList: IUser[]
+  ) => void;
+  handleLogOut: () => void;
+  currentUser: IUser | "Guest";
+}
+
+const notify = () => toast("Please sign in to add a resource");
+
+export default function Navbar({
+  handleUserClicked,
+  handleLogOut,
+  currentUser,
+}: NavProps): JSX.Element {
+  const { usersList } = useFetchUsers();
+  return (
+    <div className="navbar">
+      <NavLink to="/" className="navbarTitle navitem">
```

Comment:
I think that class doesn't follow conventions (lowercase, separated by hyphen etc)(?)

Reply:
thank you! we will implement that next time!